Lib/typing.py

```diff
@@ -2228,16 +2223,16 @@ def get_origin(tp):
     """Get the unsubscripted version of a type.
 
     This supports generic types, Callable, Tuple, Union, Literal, Final, ClassVar
-    and Annotated. Return None for unsupported types. Examples::
-
-        get_origin(Literal[42]) is Literal
-        get_origin(int) is None
-        get_origin(ClassVar[int]) is ClassVar
-        get_origin(Generic) is Generic
-        get_origin(Generic[T]) is Generic
-        get_origin(Union[T, int]) is Union
-        get_origin(List[Tuple[T, T]][int]) == list
-        get_origin(P.args) is P
+    Annotated, and others. Return None for unsupported types. Examples:
```

Comment:
Why do you remove `::` here?

Reply:
In general, I wasn't sure if there was any benefit to using `::` over `:` in a CPython docstring, since for CPython, the docstrings aren't ever rendered by Sphinx anywhere. We already have several docstrings that just use `:` in typing.py, and several that use `::`. At one point while working on this PR, I started changing them all to `:`, but then I decided that it probably would have just been unnecessary churn, so reverted most of those changes.

Reply:
Cc. docs experts @AA-Turner and @CAM-Gerlach — is there any advantage to using :: in a CPython docstring, that you know of? :)

Reply:
With `::`

<img width="1352" alt="Снимок экрана 2023-06-05 в 11 28 31" src="https://github.com/python/cpython/assets/4660275/8ce9891d-1f69-4217-b556-7e8e4d2c85b7">

With `:`

<img width="643" alt="Снимок экрана 2023-06-05 в 11 31 47" src="https://github.com/python/cpython/assets/4660275/930309c5-d4e5-464a-91e9-4a440d2aa12c">

Reply:
Thanks, that's pretty persuasive!

Reply:
```suggestion
    Annotated, and others. Return None for unsupported types. Examples::
```